### ISSUE TITLE
We need to handle 404 errors 

### ISSUE THREAD

**Issue by robertkibet** [credmark/credmark-terminal#33]
We already have the functionality in place, but we just need a nice looking page design for it, right now it will show you an empty white screen page when you hit a 404 error.
@MattCMK

See screenshot below.

![image.png](https://images.zenhubusercontent.com/62b57f6a317b342a8f9507de/673618e2-b484-4631-a1bb-a29c515dbbd0)

**Comment by MattCMK** [comment]
@robertkibet I will speak with Mark - but it may take some time - are you aware of any free to use designs?

**Comment by robertkibet** [comment]
> @robertkibet I will speak with Mark - but it may take some time - are you aware of any free to use designs? The simplest way would be something like a text box and say "Oh sorry - something went wrong (404 error)" and maybe use the green tone for the color so that you see it better on the gray background

yes, no worries, I can make a simple 404 page and improve it in future @MattCMK
